client/src/modules/profiles/pages/userProfile.tsx

```diff
@@ -21,7 +21,7 @@ export const UserProfilePage = () => {
   const logout = useLogout();
   const router = useRouter();
 
-  const confirmDelete = useConfirmDelete({ doubleConfirm: true });
+  const confirmDelete = useConfirmDelete({ buttonText: 'Yes, im sure' });
```

Comment:
```suggestion
  const confirmDelete = useConfirmDelete({
    body: 'Are you sure you want to delete your account? Account deletion cannot be reversed.',
    buttonText: 'Delete account',
  });
```
Could we change at the same time the default text? And maybe also make explicit that is account what is being deleted.